addon/styles/addon.scss

```diff
@@ -185,6 +186,10 @@
 	margin-right: auto;
 }
 
+.frost-list .container .vertical-item {
+  min-height: 30px;
```

Comment:
Since a regression in current version of SM. min-height is mandatory if you're not specify `alwaysUseDefaultHeight=true` for your {{vertical-collection}}. The current list height for our use case is default to 60px for small view, but I'm not sure if less height list-item will be used by anyone. 30px is basically a random pick small value.